README.md

```diff
@@ -135,18 +135,13 @@ INFO[0029] READY. Run `lima` to open the shell.
 Linux
 ```
 
-#### start an instance automatically
-
-```console
-limactl start --tty=false 
-```
-
 Detailed usage:
 
-- Run `limactl start <INSTANCE>` to start the Linux instance.
+- Run `limactl start <INSTANCE> [--tty=false]` to start the Linux instance.
   The default instance name is "default".
   Lima automatically opens an editor (`vi`) for reviewing and modifying the configuration.
   Wait until "READY" to be printed on the host terminal.
+  `--tty=false` disables the interactive prompt.
```

Comment:
We should change `Lima automatically opens an editor` to `Lima offers to open an editor` above, and then write:

```suggestion
  `--tty=false` disables the interactive prompt to open an editor.
```

To make things more explicit for somebody just reading the `README` who hasn't actually run `limactl` before.